src/services/ogc-features/src/main/java/org/geotools/data/postgis/SchemaUnawarePostGISDialect.java

```diff
@@ -1,12 +1,12 @@
-package com.camptocamp.opendata.ogc.features.autoconfigure.geotools;
+package org.geotools.data.postgis;
```

Comment:
Cause otherwise I'd need to re-write it all. There are package protected methods I can't override. It's explained in the commit ```commit 554391bfd239d238b1c640e6802dea4b25b11f81
Author: Gabriel Roldan <gabriel.roldan@camptocamp.com>
Date:   Thu Aug 22 19:29:47 2024 -0300

    Support postgis schema names with apostrophes
    
    Move SchemaUnawarePostGISDialect to package org.geotools.data.postgis
    to be able of overridding the package private method
    lookupGeometryType()```

Reply:
ok sorry for having missed a commit-by-commit approach, makes sense